MAUI/DataGrid/selection.md

```diff
@@ -324,6 +332,21 @@ By default, the current cell is focusable for all the columns. To disable curren
 {% endhighlight %}
 {% endtabs %}
 
+## Disable Deleting the rows
+
+By default, the selected rows will be deleted. If we set the `DataGrid.AllowDeleting` property to `False`, the deletion of rows will be disabled.
```

Comment:
By default, the selected rows will be deleted when pressing the `delete` key. It can be disabled by setting the `SfDataGrid.AllowDeleting` property to `false`.

Reply:
Updated